lib/components/Widgets/Details/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { Badge } from "@/components/Information/Badge"
+import { ComponentProps, forwardRef } from "react"
+import { DetailsItem, DetailsItemType } from "../DetailsItem"
+import { Weekdays } from "../Weekdays"
+
+interface DetailsType {
+  title?: string
+  details?: DetailsItemType[]
+  manager?: { title: string; name: string; avatar: string }
+  teams?: { title: string; list: string[] }
+  workableDays?: {
+    title: string
+    activatedDays?: ComponentProps<typeof Weekdays>["activatedDays"]
+  }
+}
+
+export const Details = forwardRef<HTMLDivElement, DetailsType>(
```

Comment:
I see this suuuuuper tied to our business model, so I don't think that Factorial One is the place for this. I see a room for each detail type, but in a generic way. Convince me otherwise

Reply:
I think you are right. I'll use this code to create this component from Factorial then and this PR to move and expose the DetailsItem component.